cloudsim-plus/src/main/java/org/cloudbus/cloudsim/power/models/PowerModelHost.java

```diff
@@ -0,0 +1,83 @@
+package org.cloudbus.cloudsim.power.models;
+
+import org.cloudbus.cloudsim.hosts.Host;
+import org.cloudbus.cloudsim.power.PowerMeasurement;
+
+/**
+ * Power model for data center hosts.
+ */
+public class PowerModelHost extends PowerModel {
+
+    public static PowerModelHost NULL = new PowerModelHost(0 ,0) {
+        @Override public PowerMeasurement getPowerMeasurement() { return new PowerMeasurement(); }
+    };
+
+    private Host host;
+
+    private boolean shared;
+    private double staticPower;
+    private double maxPower;
+    private double wattPerMips;
+
+    /**
+     * Instantiates a {@link PowerModelHost} by specifying its static and max power usage.
+     *
+     * @param maxPower power (in watts) the host consumes under full load.
+     * @param staticPower power (in watts) the host consumes when idle.
+     */
+    public PowerModelHost(final double staticPower, final double maxPower) {
+        this.shared = false;
+        this.staticPower = staticPower;
+        this.maxPower = maxPower;
+    }
+
+    /**
+     * Instantiates a shared {@link PowerModelHost} by specifying its watt per MIPS.
+     *
+     * @param wattPerMips Incremental watt per MIPS the host consumes under load.
+     */
+    public PowerModelHost(final double wattPerMips) {
+        this.shared = true;
```

Comment:
What is a shared PowerModelHost?

Reply:
It's a host that is used by other entities outside of the simulation scope (in which case you compute power usage differently).

This is already a little coupled to LEAF, thank's for catching this, this shouldn't be part of the core library!